src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/arrivals/ArrivalsService.kt

```diff
@@ -109,50 +110,50 @@ class ArrivalsService(
     moveId: String,
     prisonNumber: String
   ): ConfirmArrivalResponse {
-    val bookingId = prisonService.admitOffenderOnNewBooking(prisonNumber, confirmArrivalDetail)
+    val inmateDetail = prisonService.admitOffenderOnNewBooking(prisonNumber, confirmArrivalDetail)
     confirmedArrivalService.add(
       movementId = moveId,
       prisonNumber = prisonNumber,
       prisonId = confirmArrivalDetail.prisonId!!,
-      bookingId = bookingId,
+      bookingId = inmateDetail.bookingId,
       arrivalDate = confirmArrivalDetail.bookingInTime?.toLocalDate() ?: LocalDate.now(clock),
       arrivalType = ArrivalType.NEW_BOOKING_EXISTING_OFFENDER
     )
-    return ConfirmArrivalResponse(offenderNo = prisonNumber)
+    return ConfirmArrivalResponse(prisonNumber = prisonNumber, location = locationFormatter.format(inmateDetail.assignedLivingUnit.description))
```

Comment:
I was suggesting you could return `prisonNumber` and `offenderNo` - both set to the same value (optionally deprecating `offenderNo`). 
Then you wouldn't break backwards compatibility and could release whenever. 
We could then alter the frontend and remove usage of `offenderNo` and once thats done, remove `offenderNo` from the backend

Reply:
done